search/intermediate_result.cpp

```diff
@@ -24,7 +24,6 @@ namespace search
 {
 
 /// All constants in meters.
```

Comment:
Comment said "All constants", not "All constants, where number of constants is greater than one."